operators/test/e2e/es/failure_test.go

```diff
@@ -74,20 +72,8 @@ func TestKillCorrectPVReuse(t *testing.T) {
 
 	k := test.NewK8sClientOrFatal()
 
-	// When working with multiple PVs on a single pod, there's a risk each PV get assigned to a different zone,
-	// not taking into consideration pod scheduling constraints. As a result, the pod becomes unschedulable.
-	// This is a Kubernetes issue, that can be dealt with relying on storage classes with `volumeBindingMode: WaitForFirstConsumer`.
-	// With this binding mode, the pod is scheduled before its PVs, which then take into account zone constraints.
-	// That's the only way to work with multiple PVs. Since the k8s cluster here may have a default storage class
-	// with `volumeBindingMode: Immediate`, we create a new one, based on the default storage class, that uses
-	// `waitForFirstConsumer`.
-	lateBinding := v1.VolumeBindingWaitForFirstConsumer
 	sc, err := elasticsearch.DefaultStorageClass(k)
```

Comment:
Do we still need this line?

Reply:
I kept it because we ask for the storage class when creating a PV. 
I can take it off.